### PR TITLE
Fix / Deployments pulling on status update

### PR DESCRIPTION
**Pull request by WilliamTraoreee**
# What does this PR do?

- Fix deployments refresh when action is launch on environment
- Change progress icon

---

### Global

- [x] This PR does not introduce any breaking change
- [ ] This PR introduces breaking change(s) and has been labeled as such
- [x] I have found someone to review this PR and pinged him

### Store

- [ ] This PR introduces new store changes

### NX

- [x] I have run the dep-graph locally and made sure the tree was clean i.e no circular dependencies
- [x] I have followed the library pattern i.e `feature`, `ui`, `data`, `utils`

### Clean Code

- [x] I made sure the code is type safe (no any)
- [ ] I have included a feature flag on my feature, if applicable

### CODE REVIEW

**Comment by evoxmusic** [comment]
A preview environment was automatically created via Qovery.  
Click on the link below to follow its deployment and use it.  
:point_right: [[PR] staging - Fix / Deployments pulling on status update - 2022-06-23T11:53:26Z](https://console.qovery.com/platform/organization/141c07c8-0dd9-4623-983b-3fdd61867255/projects/977bd069-20f7-4b24-bd95-28b44174cf43/environments/e37acf61-a0d2-43e8-9c23-1188b0f9a1bb/applications)

File: libs/pages/services/src/lib/feature/page-deployments-feature/page-deployments-feature.tsx [hunk 1]
```diff
@@ -65,13 +65,12 @@ export function PageDeploymentsFeature() {
 
     !environment?.deployments && fetchEnv()
 
-    const pullDeployments = setInterval(() => {
-      dispatch(fetchEnvironmentDeploymentHistory({ environmentId, silently: true }))
-    }, 3000)
+    const pullDeployments = setInterval(
+      () => dispatch(fetchEnvironmentDeploymentHistory({ environmentId, silently: true })),
+      2500
```
Comment (RemiBonnet) [inline libs/pages/services/src/lib/feature/page-deployments-feature/page-deployments-feature.tsx:70]: Why we use `2500` for this and not `3000` ?

**Review by RemiBonnet — CHANGES_REQUESTED** [review]
Hey @WilliamTraoreee, thanks !

I think all fetch (outside the status) need an condition because here you fetch element, that we don't need on the page

**Pull request merged**